sass/base/components/navigation/feature-nav-open.twig

```diff
@@ -1,5 +1,5 @@
 <div class="header__inner-wrapper">
-  <nav style="top:-10px" role="navigation" aria-labelledby="block-campaign-base-main-menu-menu" id="block-campaign-base-main-menu" class="contextual-region block block-menu navigation menu--main crNavigation-processed show">          
+  <nav style="top:-10px" role="navigation" aria-labelledby="block-campaign-base-main-menu-menu" id="block-campaign-base-main-menu" class="contextual-region block block-menu navigation menu--main feature-main-nav crNavigation-processed show">          
```

Comment:
Are we able to remove this inline styling and put it in the SCSS, while we're here?

I _seem_ to remember this was a bugfix required because of how certain CSS wasn't loaded straight-away, possibly due to classes being updated with JS or something. Kinda feels like if we just target the nav element with no other classes, it'd have the same effect, but would be able to be override easily, should we need to. And keeps crummy inline styles out of the markup.